DevExpress.Mvvm.CodeGenerators/Generators/PropertyGenerator.cs

```diff
@@ -52,18 +71,27 @@ static class PropertyGenerator {
             return propertyName;
         }
 
-        static void AppendRaiseChangingMethod(SourceBuilder source, ChangeEventRaiseMode? eventRaiseMode, string propertyName) {
-            if(eventRaiseMode == ChangeEventRaiseMode.EventArgs)
-                source.Append("RaisePropertyChanging(").Append(propertyName).AppendLine("ChangingEventArgs);");
-            if(eventRaiseMode == ChangeEventRaiseMode.PropertyName)
-                source.Append("RaisePropertyChanging(nameof(").Append(propertyName).AppendLine("));");
-        }
-
-        static void AppendRaiseChangedMethod(SourceBuilder source, ChangeEventRaiseMode? eventRaiseMode, string propertyName) {
-            if(eventRaiseMode == ChangeEventRaiseMode.EventArgs)
-                source.Append("RaisePropertyChanged(").Append(propertyName).AppendLine("ChangedEventArgs);");
-            if(eventRaiseMode == ChangeEventRaiseMode.PropertyName)
-                source.Append("RaisePropertyChanged(nameof(").Append(propertyName).AppendLine("));");
+        static void AppendRaiseChangeMethod(SourceBuilder source, RaiseInfo? raiseInfo, string propertyName, string suffix) {
+            if(raiseInfo?.Mode == ChangeEventRaiseMode.EventArgs) {
+                source
+                    .Append(raiseInfo.Value.Prefix.ToStringValue())
+                    .Append("PropertyChang")
+                    .Append(suffix)
+                    .Append('(')
+                    .Append(propertyName)
+                    .Append("Chang")
+                    .Append(suffix)
```

Comment:
"Append" calls are chained to avoid excessive memory traffci and improve performance. Correct approach is to use [Interpolated String Handler](https://docs.microsoft.com/en-us/dotnet/csharp/whats-new/tutorials/interpolated-string-handler) available in C# 10. Probably somewhere in the future.